src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -964,55 +1056,63 @@ public void clickRemoveRubricColLinkAndConfirm(int qnIndex, int col) {
     }
 
     public void verifyRankOptionIsHiddenForNewQuestion(int optionIndex) {
-        WebElement optionBox = browser.driver.findElement(By.id("rankOption-" + optionIndex + "--1"));
+        WebElement optionBox = browser.driver.findElement(By.id("rankOption-" + optionIndex + "-" + NEW_QUESTION_NUM));
         assertFalse(optionBox.isDisplayed());
     }
     
-    public void fillRankOptionForNewQuestion(int optionIndex, String optionText) {
-        WebElement optionBox = browser.driver.findElement(By.id("rankOption-" + optionIndex + "--1"));
-        fillTextBox(optionBox, optionText);
-    }
-    
     public void fillRankOptionForQuestion(int qnIndx, int optionIndex, String optionText) {
         WebElement optionBox = browser.driver.findElement(By.id("rankOption-" + optionIndex + "-" + qnIndx));
         fillTextBox(optionBox, optionText);
     }
     
-    public void tickDuplicatesAllowedCheckboxForNewQuestion() {
-        tickDuplicatesAllowedCheckboxForQuestion(-1);
+    public void fillRankOptionForNewQuestion(int optionIndex, String optionText) {
+        fillRankOptionForQuestion(NEW_QUESTION_NUM, optionIndex, optionText);
     }
     
     public void tickDuplicatesAllowedCheckboxForQuestion(int qnIndex) {
         WebElement checkBox = toggleDuplicatesAllowedCheckBox(qnIndex);
         assertTrue(checkBox.isSelected());
     }
     
+    public void tickDuplicatesAllowedCheckboxForNewQuestion() {
+        WebElement checkBox = toggleDuplicatesAllowedCheckBoxForNewQuestion();
+        assertTrue(checkBox.isSelected());
+    }
+    
```

Comment:
tickDuplicatesAllowedCheckboxForNewQuestion can possibly call tickDuplicatesAllowedCheckboxForQuestion instead, since they share similar behaviour. Then, toggleDuplicatesAllowedCheckBoxForNewQuestion can be possibly removed, which may make this part easier to comprehend.